storefront/app/helpers/workarea/storefront/navigation_helper.rb

```diff
@@ -65,22 +65,25 @@ def link_to_menu(menu)
         end
       end
 
-      def storefront_path_for(taxon)
-        if taxon.url?
-          taxon.url
-        elsif taxon.root?
+      def storefront_path_for(model)
```

Comment:
This wasn't intended to allow passing any model to it. We should correct it upstream. Where are we calling this without a taxon?